qpylib/log_qpylib.py

```diff
@@ -83,21 +83,22 @@ def _default_log_level():
 def _log_file_location():
     return app_qpylib.get_log_path('app.log')
 
-def _generate_handlers():
+def _generate_handlers(syslog_enabled=True):
```

Comment:
No need for `=True` here, because `_generate_handlers` will always receive an argument.